src/admin-views/attendees.php

```diff
@@ -11,19 +11,31 @@
 $singular = $pto->labels->singular_name;
 
 /**
- * Whether or not we should display attendees title
+ * Whether we should display the "Attendees" title.
  *
  * @since  4.6.2
+ * @since  TBD Append the post ID to the Attendees page title and each Ticket's name.
  *
- * @param  boolean                          $show_title
- * @param  Tribe__Tickets__Tickets_Handler  $handler
+ * @param boolean                         $show_title
+ * @param Tribe__Tickets__Tickets_Handler $handler
  */
 $show_title = apply_filters( 'tribe_tickets_attendees_show_title', true, $attendees );
 ?>
 
 <div class="wrap tribe-report-page">
 	<?php if ( $show_title ) : ?>
-		<h1><?php esc_html_e( 'Attendees', 'event-tickets' ); ?></h1>
+		<h1>
+			<?php
+			echo esc_html(
+			// Translators: 1: the post title, 2: the post ID.
+				sprintf(
+					_x( 'Attendees for: %1$s [#%2$d]', 'event-tickets' ),
```

Comment:
:no_entry_sign: A gettext call containing placeholders was found, but was not accompanied by a "translators:" comment on the line above to clarify the meaning of the placeholders.

<sub>hash: 8612ad0f6e9758fa6b2be0f09929c26c</sub>